src/main/java/net/snowflake/ingest/streaming/internal/ConfigureRequest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import net.snowflake.ingest.utils.Utils;
+
+/** Class used to serialize the client / channel configure request. */
+public class ConfigureRequest implements StreamingIngestRequest {
+  @JsonProperty("role")
+  private String role;
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  @JsonProperty("database")
+  private String database;
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  @JsonProperty("schema")
+  private String schema;
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  @JsonProperty("table")
+  private String table;
```

Comment:
ClientConfigure does not use these fields, not sure why they're here? If you're thinking we should use one configure requests for both client configure and channel configure, then IMO these should be completely separate contracts. there is no need to have a common contract for these two APIs.

I believe i left a similar comment in the last round but might be mistaken, will review old comments once im done iwth a pass here.

Reply:
Created `ClientConfigureRequest` and `ChannelConfigureRequest`